content/en/docs/concepts/scheduling-eviction/eviction-policy.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Eviction Policy
+content_template: templates/concept
+weight: 50
+---
+
+<!-- overview -->
+
+This page is an overview of Kubernetes' policy for eviction.
+
+<!-- body -->
+
+## Eviction Policy
+
+The `kubelet` can proactively monitor for and prevent total starvation of a
```

Comment:
Consider using a glossary tooltip for the first use of kubelet here: see https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#glossary for details.

Reply:
sure let me take a look at it.

Reply:
i have added tool tip for first use.